std/bitmanip.d

```diff
@@ -300,7 +300,7 @@ The example above creates a tagged reference to an Object in the struct A. This
 as $(D taggedPointer), except the first argument which must be a class type instead of a pointer type.
 
 $(RED Warning: Don't use $(D taggedClassRef) with references to garbage collected objects, as it will result in
-undefined behaviour. See $(LINK http://dlang.org/garbage.html) for details.)
+undefined behaviour. See $(LINK ../spec/garbage.html) for details.)
```

Comment:
As I mentioned above, we also have the option of defining a macro that will be defined differently for DDox. So, for example, a `$(PHOBOS2SPEC ...)` macro could be defined to be a relative link to `../spec/$1` by default but redifined to an absolute URL for DDox.